pkg/storj/encryption.go

```diff
@@ -3,6 +3,13 @@
 
 package storj
 
+import (
+	"github.com/zeebo/errs"
+)
+
+// ErrKey is used when something goes wrong a key.
+var ErrKey = errs.Class("key")
```

Comment:
This is not used in the latest change. So let's remove it. We can bring it back when necessary.

Reply:
Ive forgotten to remove it